ts/sagas/instabug.ts

```diff
@@ -26,6 +26,13 @@ function* watchInstabugSaga(): IterableIterator<Effect> {
   }
 }
 
+export function* updateInstabugBadgeSaga(): IterableIterator<Effect> {
```

Comment:
```suggestion 
function* updateInstabugBadgeSaga(): IterableIterator<Effect> {
```
in **watchInstabugSaga** you could replace the same instruction of **updateInstabugBadgeSaga**
with `yield call(updateInstabugBadgeSaga);`

Reply:
Ok